main.go

```diff
@@ -223,7 +223,8 @@ func mainLoop(oracleInstance *oracle.Oracle, onchain *oracle.Onchain, cfg *oracl
 	lastReconciliationTime := int64(0)
 
 	// Load all the validators from the beacon chain
-	onchain.RefreshBeaconValidators()
+	// TODO: see what API endpoints are affected by commenting this
+	// onchain.RefreshBeaconValidators()
```

Comment:
Not sure this will work fine if we comment this. I mean, we can't leave it, otherwise it wont be deterministic, but commenting it can lead to the following. If you start the oracle, you won't have any validator loaded until UpdateValidatorsIntervalSlots have passed, and that is dangerous imho.

Reply:
We can leave it uncommented & as it is now if we dont depend on the validators loaded by `onchain` object when doing the cleanup! Oracle could fetch the validators onchain state directoly, without asking  `onchain`